### PR TITLE
[JSON] rewordings consistency and clarity

### PR DESCRIPTION
**Pull request by fernyrepos**
Screen Shot 2023-09-05 at 10 57 31 AM

old blockset

Screen Shot 2023-09-05 at 11 01 07 AM

new blockset

- made booleans internally consistent with missing question marks
- removed "get" where applicable
- made some blocks be more internally consistent when it comes to value order, like putting the array before the new value
- removed unnecessary words like "content" in the list block

### CODE REVIEW

**Comment by fernyrepos** [comment]
(pr also includes the previously merged commit with the change to the replace block cause i didn't update my branch)

File: extensions/Skyhigh173/json.js [hunk 1]
```diff
@@ -163,7 +163,7 @@
           {
             opcode: "json_set",
             blockType: Scratch.BlockType.REPORTER,
-            text: "set [item] to [value] in [json]",
+            text: "set [item] in [json] to [value]",
```
[the remark this answers is not in the document]
Comment (fernyrepos) [reply extensions/Skyhigh173/json.js:166]: in other instances in the extension (and most instances in blocks in general) the thing you wanna set is typically at the end. 

examples:
Screen Shot 2023-09-05 at 12 46 12 PM

its a tiny thing for sure but its good for tiny little processes your brain can skip with enough consistency

**Comment by CubesterYT** [comment]
I feel like the question marks should stay, if we were to follow scratch block design. All booleans I've seen in scratch end in a question mark usually.

**Comment by CubesterYT** [comment]
Oh shoot disregard my previous message, I read the PR wrong.

**Pull request merged**